core/src/main/java/org/elasticsearch/action/index/IndexRequest.java

```diff
@@ -49,14 +49,14 @@
 /**
  * Index request to index a typed JSON document into a specific index and make it searchable. Best
  * created using {@link org.elasticsearch.client.Requests#indexRequest(String)}.
- * <p>
```

Comment:
I think we should keep these HTML markups?  So that the javadocs when viewed in a browser look correct?  They are annoying though...